features/step_definitions/project_version_steps.rb

```diff
@@ -0,0 +1,16 @@
+Given /^project version (.+) exists for project (.+)(?: with creation date set (?:(1 day|(?:[2-9]|\d{2,}) days) ago))$/ do |name,project_name,nb_days|
+  options = {
+    name: name.downcase.gsub(/\s+/, '-'),
```

Comment:
I would not transform the version name. There are currently no restrictions on version characters and no reason to put it in lowercase or replace spaces.